spec/lib/mercury/mercury_spec.rb

```diff
@@ -29,7 +29,7 @@
       em do
         Mercury.open do |m|
           m.close do
-            expect { m.publish(queue, {'a' => 1}) }.to raise_error /closed/
+            expect { m.publish(queue, 'a' => 1) }.to raise_error /closed/
```

Comment:
Braces are more appropriate here. Omitting braces around last-argument hash literals is a hack for named arguments. Here, I'm legitimately passing a data structure to `publish`, so I want to see that structure. Most new code we write should be using Ruby 2's keyword arguments anyway.